roundrect/ImageGeneration.swift

```diff
@@ -8,6 +8,36 @@
 
 import UIKit
 
+public enum Rounding: Equatable {
+  case all(CGFloat)
+  case some(corners: UIRectCorner, radii: CGSize)
```

Comment:
Why `CGFloat` for `.all`, and `CGSize` for `.some`? I'm wondering because you are converting this below as well anyways.

Reply:
`CGFloat` just gets turned into a `CGSize` downstream

Reply:
Its mostly a convenience